pom.xml

```diff
@@ -15,13 +39,43 @@
                     <target>1.8</target>
```

Comment:
sync sdk is on java7, I don't know about changefeed processor. But if changefeed processor doesn't require 8, you can go with 7 here.

Reply:
targeting 1.8  is ok for this library.